src/common/Pagination.tsx

```diff
@@ -52,6 +52,8 @@ const Pagination = ({ totalPosts, limit, page, setPage, currentChannelId }: Ipag
 export default Pagination;
 
 const Button = styled.button<IButtonStyle>`
-  background-color: ${({ page, i }) => (page === i + 1 ? 'white' : null)};
-  cursor: pointer;
+  &[currentPage] {
+    border: 2px solid red;
+  }
+  background-color: ${({ page, i }) => (page === i + 1 ? 'red' : null)};
```

Comment:
p4;
해당 소스코드로 수정하셔도 동작할 것 같습니다!
```suggestion
  background-color: ${({ page, i }) => (page === i + 1 && 'red')};
```